README.md

```diff
@@ -1,6 +1,6 @@
 # xhr
 
-A small xhr wrapper. Designed for use with [browserify](http://browserify.org/).
+A small XMLHttpRequest wrapper. Designed for use with [browserify](http://browserify.org/).
```

Comment:
I wouldn't be against it.  I was mostly just aiming to reduce the name confusion with this pull request (there's already a ton of links to XMLHttpRequest docs/specs throughout the readme).